input/resources/operationdefinition-knowledgeartifact-approve.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OperationDefinition xmlns="http://hl7.org/fhir">
+    <id value="KnowledgeArtifact-approve"/>
+    <url value="http://hl7.org/fhir/uv/cmi/OperationDefinition/KnowledgeArtifact-approve"/>
+    <version value="1.0.0"/>
+    <name value="KnowledgeArtifactApprove"/>
+    <title value="Knowledge Artifact Approve"/>
+    <status value="draft"/>
+    <kind value="operation"/>
+    <date value="2023-02-24"/>
+    <publisher value="HL7 (FHIR Project)"/>
+    <contact>
+        <telecom>
+            <system value="url"/>
+            <value value="http://hl7.org/fhir"/>
+        </telecom>
+        <telecom>
+            <system value="email"/>
+            <value value="fhir@lists.hl7.org"/>
+        </telecom>
+    </contact>
+    <description value="The approve operation supports applying an approval to an existing artifact, regardless of status. The operation sets the _date_ and _approvalDate_ elements of the approved artifact, and is otherwise only allowed to create ArtifactAssessment resources in the repository."/>    
+    <code value="knowledge-artifact-approve"/>
+    <comment value="The approve operation supports the ability of a repository to record information about its' eligibility for release in an ArtifactAssessment resource upon completion of the review process. The ArtifactAssessments which are added by the operation must reference a version of the artifact."/>
```

Comment:
Need to remove apostrophe in "its' eligibility". Also, approval isn't necessarily release-specific so I'm not sure it makes sense to mention release here?

Reply:
I updated this to 

`The approve operation supports the ability of a repository to record commentary on a specific state of an artifact in an ArtifactAssessment resource by applying an approval.` 

Is that better? I wasn't sure how to distinguish the comment from the description otherwise, they ended up being identical after removing the release-specific language

Reply:
Looks good.